deploy/docker_push.sh

```diff
@@ -33,6 +33,9 @@
 # PROD_MODE: If false then pushed tags will include a -dev suffix.
 #                  Defaults to false
 
+export DOCKER_NAME=$1
+export DOCKER_IMAGE=$2
+export IMAGE_VERSION=$3
 export tmp_tag="$(date +%Y-%m-%d)"
```

Comment:
```suggestion
DOCKER_NAME=$1
DOCKER_IMAGE=$2
IMAGE_VERSION=$3
tmp_tag="$(date +%Y-%m-%d)"
```

I'm sure there is no need to export these variables.

Reply:
you're right, adjusted.